doc/assets/js/theme.js

```diff
@@ -97,17 +108,55 @@ function addTabs(){
     if (window.sessionStorage) {
       window.sessionStorage.setItem('tabActive', key)
     }
+
+    return true
   }
 
-  function getTabsByKey(key) {
-    return [...document.querySelectorAll(`[data-tab="${key}"]`)]
+  const tabs = [...document.querySelectorAll('.tabs li')]
+  const tabContent = [...document.querySelectorAll('.tab-content section')]
```

Comment:
Is this Array casting really necessary? You're using only `forEach()` which should be available in `NodeList` as well.